### DISCUSSION TITLE
How to set up openSnitch for newbies

### DISCUSSION THREAD

**Discussion by proggaliu** [evilsocket/opensnitch#556] [Q&A] [answered]
I'm completely new to Ubuntu so I'm pretty sure I just don't know what I'm doing. Anyhow I installed openSnitch GUI ver 1.4.2 and get the logo in the taskbar. I can click on it and open "OpenSnitch Network Statistics 1.4.2" window. The issue is that it does not detect any network connections.

Any help would be much appreciated :)

**Comment by gustavo-iniguez-goya** [comment, marked as answer]
Hi @proggaliu ,

You need to install the daemon: https://github.com/evilsocket/opensnitch/releases/tag/v1.4.3
Download the deb package for your architecture, and it'll start intercepting connections.

**Comment by proggaliu** [comment]
Thank you, works now :D
